src/m365/flow/commands/flow-get.ts

```diff
@@ -46,12 +56,14 @@ class FlowGetCommand extends AzmgmtCommand {
     request
       .get(requestOptions)
       .then((res: any): void => {
```

Comment:
I've added a missing interface `Flow` here so that we don't need to use `any` and can benefit from design-time typesafety.